<launch>
<!-- Compare mesh between two meshes -->
    <arg name="visualization" default="true"/>
    <node pkg="tf" type="static_transform_publisher" name="world_map" args="0 0 0 0 0 0 world map 10"/>
    <node pkg="adaptive_viewutility" type="test_gridmap_loader" name="test_mesh" output="screen">
        <param name="original_map_path" value="$(find adaptive_viewutility)/../resources/airsim_groundtruth.obj"/>
        <param name="saved_map_path" value="$(find adaptive_viewutility)/../output/saved_map.bag"/>
    </node>

    <node pkg="grid_map_visualization" type="grid_map_visualization" name="grid_map_visualization" output="screen">
        <rosparam command="load" file="$(find adaptive_viewutility)/launch/visualize.yaml" />
    </node>

    <group if="$(arg visualization)">
        <node type="rviz" name="rviz" pkg="rviz" args="-d $(find adaptive_viewutility)/launch/config_loader.rviz" />
    </group>
</launch>
